src/components/Integrations/EmptyState.tsx

```diff
@@ -0,0 +1,119 @@
+import {
+  Button,
+  ButtonVariant,
+  Card,
+  CardBody,
+  CardHeader,
+  EmptyState,
+  EmptyStateBody,
+  EmptyStateIcon,
+  Gallery,
+  Icon,
+  Text,
+  TextContent,
+  TextVariants,
+  Title,
+  TitleSizes,
+} from '@patternfly/react-core';
+import {
+  ArrowRightIcon,
+  CogIcon,
+  CubesIcon,
+  HelpIcon,
+  InfrastructureIcon,
+} from '@patternfly/react-icons';
+import React from 'react';
+
+import { Messages } from '../../properties/Messages';
+import { Link } from 'react-router-dom';
+import { useAppContext } from '../../app/AppContext';
+
+const getEmptyIntegrationCard = (props) => {
+  const { TitleIcon, title, body, link, isOrgAdmin } = props;
```

Comment:
I'd spread the props directly when using the `props`. Also the name is not ideal, it does not indicate it's going to produce React component.
```suggestion
const EmptyIntegrationCard = ({ TitleIcon, title, body, link, isOrgAdmin }) => {
```

Reply:
@karelhala good catch. Done!